src/main/java/com/auctionapp/api/model/entities/User.java

```diff
@@ -0,0 +1,46 @@
+package com.auctionapp.api.model.entities;
+
+import java.sql.Timestamp;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Table;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "users")
+@Getter
+@Setter
+@NoArgsConstructor
```

Comment:
Please no. Make explicit getters, setters, constructors, equals, hashCode, and toString methods (all as needed, obviously), no need for someone to comb through annotations to see what is there and what is not. I'd remove all lombok usages - I don't think it should be necessary for this project.